fix: LEAP-461: Fix displaying conditions for comments tab

robot-ci-heartex opened the pull request:
Hi @Gondragos!

This PR was [created](https://github.com/HumanSignal/label-studio/actions/runs/7340063163) in a response to PRs in upstream repos:
- https://github.com/HumanSignal/label-studio-frontend/pull/1650

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7340063163

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7340418839

hlomzik committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7424658505

hlomzik committed:
Merge branch 'origin/develop' into 'fb-leap-461/comments'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7424658505

hlomzik committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7424737794

Gondragos commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/7463082236)
> **Error**: Merge conflict detected, please resolve it using the command line.

Gondragos committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7463171409

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7463171409

Gondragos committed:
Merge branch 'origin/develop' into 'fb-leap-461/comments'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7463171409

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7463221301

Commit:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7465595583

Commit:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7465595583

Commit:
Merge branch 'origin/develop' into 'fb-leap-461/comments'
